config/settings.py

```diff
@@ -13,6 +13,8 @@
 import os
 from pathlib import Path
 from django.utils.translation import gettext_lazy as _
+from dotenv import load_dotenv
```

Comment:
You don't need dotenv, python works fine for env variables. use DESIRED_KEY=os.getenv('DESIRED_KEY')

Reply:
Okay, thanks for the feedback
I'm getting it out now